modules/xcursor.nix

```diff
@@ -54,7 +61,7 @@ in
     home.packages = [cfg.package];
 
     xsession.initExtra = ''
-      ${pkgs.xorg.xsetroot}/bin/xsetroot -xcf ${cfg.package}/share/icons/${cfg.name}/cursors/X_cursor ${toString cfg.size}
+      ${pkgs.xorg.xsetroot}/bin/xsetroot -xcf ${cfg.package}/share/icons/${cfg.name}/cursors/${cfg.cursor} ${toString cfg.size}
```

Comment:
I wonder whether the option is necessary. Is there any strong desire to have something other than `left_ptr` as the root cursor?

@league What do you think?